Tutorials/Tutorial08_Tessellation/assets/terrain.hsh

```diff
@@ -71,7 +71,7 @@ TerrainHSConstFuncOut ConstantHS( InputPatch<TerrainVSOut, 1> inputPatch/*, uint
 [outputtopology("triangle_ccw")]
 [outputcontrolpoints(1)]
 [patchconstantfunc("ConstantHS")]
-[maxtessfactor( (float)(BLOCK_SIZE) )]
```

Comment:
Why did you need to change this? As far as I can remember, this was required to make it work on Android or something like this.

Reply:
DXC can't compile that